site/en/install/lang_java.md

```diff
@@ -1,207 +1,203 @@
-# Install TensorFlow for Java
+# Install TensorFlow Java
 
-TensorFlow provides a
-[Java API](https://www.tensorflow.org/api_docs/java/reference/org/tensorflow/package-summary)—
-particularly useful for loading models created with Python and running them
-within a Java application.
+[TensorFlow Java](https://github.com/tensorflow/java) can run on any JVM for building, training and 
+deploying machine learning models. It supports both CPU and GPU execution, in graph or eager mode, and
+presents a rich API for using TensorFlow in a JVM environment. Java and other JVM languages, like Scala
+and Kotlin, are frequently used in small-to-large enterprises all over the world, which makes TensorFlow 
+a strategic choice for adopting machine learning at a large scale.
 
 Caution: The TensorFlow Java API is *not* covered by the TensorFlow
 [API stability guarantees](../guide/versions.md).
 
-## Nightly Libtensorflow Java packages
 
-Libtensorflow JNI packages are built nightly and uploaded to GCS for all
-supported platforms. They are uploaded to the
-[libtensorflow-nightly GCS bucket](https://storage.googleapis.com/libtensorflow-nightly)
-and are indexed by operating system and date built.
+## Requirements
 
-## Supported Platforms
-
-TensorFlow for Java is supported on the following systems:
+TensorFlow Java runs on Java 8 and above, and supports out-of-the-box the following platforms:
 
 * Ubuntu 16.04 or higher; 64-bit, x86
-* macOS 10.12.6 (Sierra) or higher
+* macOS 10.12.6 (Sierra) or higher; 64-bit, x86
 * Windows 7 or higher; 64-bit, x86
 
-To use TensorFlow on Android see [TensorFlow Lite](https://tensorflow.org/lite)
+*Note: To use TensorFlow on Android, see [TensorFlow Lite](https://tensorflow.org/lite)*
+
+
+## Versions
 
-## TensorFlow with Apache Maven
+TensorFlow Java has its own release cycle, independent from the [TensorFlow runtime](https://github.com/tensorflow/tensorflow).
+In consequence, its version does not match the version of TensorFlow runtime it runs on. Consult the TensorFlow Java 
+[versioning table](https://github.com/tensorflow/java/#tensorflow-version-support) to list all versions available 
+and their mapping with the TensorFlow runtime.
 
-To use TensorFlow with [Apache Maven](https://maven.apache.org){:.external},
-add the dependency to the project's `pom.xml` file:
+
+## Installing with Maven
+
+To include TensorFlow in your [Maven](http://maven.apache.org) application, you first need to add in
```

Comment:
"need to add to", though we could rephrase this sentence a few different ways. The "file a dependency" bit later on is a bit weird, so I'm trying to work around that.

Reply:
I've restored pretty much the phrasing used in the legacy version, I guess that should do it.